src/query/db_accessor.hpp

```diff
@@ -358,18 +438,44 @@ class DbAccessor final {
   storage::ConstraintsInfo ListAllConstraints() const { return accessor_->ListAllConstraints(); }
 };
 
-}  // namespace memgraph::query
+class SubgraphDbAccessor final {
+  DbAccessor *db_accessor_;
+  Graph *graph_;
```

Comment:
Should this be accessible publicly?

Reply:
What do you mean? Should it be public or shouldn't?

Reply:
These look like private variables, and I assume you wanted them to be since they have default visibility scope. I am not sure if we should write explicitly

```suggestion
private:
  DbAccessor *db_accessor_;
  Graph *graph_;
```

Reply:
Resolving this comment as we discussed that this is how it is used elsewhere in code